package/MDAnalysis/core/groups.py

```diff
@@ -1743,24 +1743,36 @@ def select_atoms(self, sel, *othersel, **selgroups):
         are not any duplicates, which can happen with complicated
         selections).
 
-        Existing :class:`AtomGroup` objects can be passed as named arguments,
-        which will then be available to the selection parser.
+        Examples
+        --------
+        All simple selection listed below support multiple arguments which are
+        implicitly combined. For example
 
-        Subselections can be grouped with parentheses.
+           >>> sel = universe.select_atoms('resname MET GLY')
 
```

Comment:
is equivalent to

   >>> sel = universe.select_atoms('resname MET or resname GLY')